db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -1611,14 +1611,19 @@ class NonBatchedOpsStressTest : public StressTest {
       }
     }
 
+    Status s;
+
+    bool prepared = false;
     PendingExpectedValue pending_expected_value =
-        shared->PreparePut(rand_column_family, rand_key);
+        shared->PreparePut(rand_column_family, rand_key, &prepared);
+    if (!prepared) {
+      pending_expected_value.PermitUnclosedPendingState();
```

Comment:
Should we continue the write here instead of returning early?

Reply:
This early return is to not writing to a key whose earlier write was being recovered.

Reply:
I was thinking we can still write to them? If recovery is not done, then the injected error will be returned. If recovery is done, then the write can succeed and we can commit the expected value update.

Reply:
Discussed offline